example/all.yaml

```diff
@@ -57,6 +57,7 @@ linter:
     - empty_constructor_bodies
     - empty_statements
     - file_names
+    - flutter_html
```

Comment:
`flutter_avoid_dart_html`? `flutter_warn_dart_html`? `flutter_use_of_html_libraries`?

This lint would be mostly about warning for use of `dart:html` (and related libraries) in a context where they won't be available (Flutter mobile).

Reply:
Awesome.  Thanks for the suggestions.  I went with the last suggestion but easily swayed...